docs/components/navigation.tsx

```diff
@@ -76,13 +70,9 @@ export default class Navigation extends React.Component<IProps> {
         </Menu>
         <h4>API reference</h4>
         <Menu className={styles.menu}>
-          {[
```

Comment:
If you want to remove these builder like syntaxes for calling the JS API, I can get onboard with that too. It would simplify our documentation somewhat!

Reply:
As discussed at lunch, I've removed all the reference documentation